docs/examples/RESTCONF/interface.json

```diff
@@ -0,0 +1 @@
+{"enabled":true,"name":"Ethernet4", "description":"restconf_test"}
```

Comment:
should this be in the `/src/restconf/` directory?

Reply:
done